### PR TITLE
pool: Remove referralCode support from Aave

### PR DESCRIPTION
**Pull request by axic**
The referral program is dead: https://docs.aave.com/developers/v/1.0/integrating-aave/referral-program

### CODE REVIEW

File: contracts/pools/AaveTempusPool.sol [hunk 1]
```diff
@@ -57,7 +54,12 @@ contract AaveTempusPool is TempusPool {
 
         // Deposit to AAVE
         IERC20(backingToken).safeIncreaseAllowance(address(aavePool), amount);
-        aavePool.deposit(address(backingToken), amount, address(this), referrer);
+        aavePool.deposit(
```
Comment (RedFox20) [inline contracts/pools/AaveTempusPool.sol:57]: Maybe keep it on one line?

File: test/utils/TempusPool.ts [hunk 2]
```diff
@@ -219,8 +219,7 @@ export class TempusPool extends ContractBase {
           depositPercent:      yieldToken.toBigNum(0.5), // fees are stored in YBT
           earlyRedeemPercent:  yieldToken.toBigNum(1.0),
           matureRedeemPercent: yieldToken.toBigNum(0.5)
-        },
```
Comment (RedFox20) [inline test/utils/TempusPool.ts:222]: Do we have any integration tests or deployment scripts that would need upgrading?

File: contracts/pools/AaveTempusPool.sol (hunk 1 above)
Comment (axic) [reply contracts/pools/AaveTempusPool.sol:57]: The linter required this.

File: test/utils/TempusPool.ts (hunk 2 above)
Comment (axic) [reply test/utils/TempusPool.ts:222]: Apparently not 😖

**Comment by RedFox20** [comment]
Then everything checks out 👍🏻

**Review by RedFox20 — DISMISSED** [review]

**Review by RedFox20 — APPROVED** [review]

**Pull request merged**